cdap-distributions/src/emr/install.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Copyright © 2015 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+#
+# Install CDAP under hadoop user for EMR
+#
+
+die() { echo "ERROR: ${*}"; exit 1; };
+
+# Fetch repo file for YUM
+curl http://repository.cask.co/centos/6/x86_64/cdap/3.0/cask.repo > /etc/yum.repos.d/cdap-3.0.repo
+rpm --import http://repository.cask.co/centos/6/x86_64/cdap/3.0/pubkey.gpg
```

Comment:
This should never fail, unless DNS is down. These files are hosted in S3 and EMR hosts have direct attachment to S3. I've added error checking, but it's extremely unlikely to ever trigger, which is a good thing.